### PR TITLE
Upgrade to Tyrus 2.0.2 

### PR DESCRIPTION
**Pull request by spericas**
Upgrade to Tyrus 2.0.2 that now uses the JPMS.
Note that Tyrus server module depends on Tyrus client, hence the change of scope in pom.

Signed-off-by: Santiago Pericasgeertsen <santiago.pericasgeertsen@oracle.com>

### CODE REVIEW

**Review by romain-grecourt — APPROVED** [review]

File: webserver/tyrus/pom.xml [hunk 1]
```diff
@@ -58,7 +58,6 @@
         <dependency>
             <groupId>org.glassfish.tyrus</groupId>
             <artifactId>tyrus-client</artifactId>
-            <scope>test</scope>
```
Comment (danielkec) [inline webserver/tyrus/pom.xml:61]: @spericas Why the client?

Comment (danielkec) [reply webserver/tyrus/pom.xml:61]: I don't think this is going to work without `requires org.glassfish.tyrus.container.grizzly.client;`

Comment (tomas-langer) [reply webserver/tyrus/pom.xml:61]: Please no. Let's talk to Tyrus to get a module that does not explicitly depend on a client

**Comment by spericas** [comment]
https://github.com/oracle/helidon/pull/4429

**Pull request merged**